maturity-level-one/src/appsettings.json

```diff
@@ -13,6 +13,6 @@
     }
   },
   "ConnectionStrings": {
-    "WorldCupDB": "Server=(localdb)\\MSSQLLocalDB;Initial Catalog=WorldCup2018;Trusted_Connection=True;"
+    "WorldCupDB": "Server=CLT-FVANRAEMDOX\\SQLEXPRESS;Initial Catalog=WorldCup2018;Trusted_Connection=True;"
```

Comment:
This should not be in the repository.

Reply:
For the sake of this sample repo, isn't using localdb ok? Would allow folks to just pull it and run it.

Reply:
I was more referring to the connectionstring that is pointing to `CLT-FVANRAEMDOX` ; `(localdb)` is fine for me.

Reply:
Ah sorry :D